src/fired-events.js

```diff
@@ -0,0 +1,29 @@
+export function eventHandler (client, type) {
```

Comment:
I prefer naming conventions that describe what the method will do - instead of things like 'clickHandler' etc. Can you rename to something like 'storeEventInSessionStorage'?